import_craft.py

```diff
@@ -364,4 +378,11 @@ def import_craft(context, craft_file_path, colliders, allow_no_material_mesh):
     lamp_object.select = True
     bpy.context.scene.objects.active = lamp_object
 
+    # Look for collider objects (i.e those name contains 'COL' or 'collider' )
```

Comment:
from what I read this comment is wrong; a mesh with no material_slot is also hidden?
I'd rather have a `is_collider` function returning a boolean telling wether an object is a collider or not. And maybe another function to test for mesh with no materials if it's not related to the collider test. That way it's easy and readable and self-documented to iterate on the objects you want:

```python
for collider in filter(is_collider, bpy.context.scene.objects):
    handle_collider(collider)
...
```